apps/spotlight/src/pages/splash/index.astro

```diff
@@ -4,67 +4,111 @@ import atjInfographics from '../../images/atj-infographics.png';
 ---
 
 <ContentLayout title="10x Access to Justice Splash Page">
-  <h1>A people-centered platform for online document assembly</h1>
-
-  <div class="videoWrapper">
-    <iframe
-      src="https://www.loom.com/embed/d958328ec69548ed99c6cd1a15af8a16?sid=e4df1738-9f51-4900-8b90-2e66c06fcc07"
-      frameborder="0"
-      webkitallowfullscreen
-      mozallowfullscreen
-      allowfullscreen
-      style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"
-    ></iframe>
+  <div class="offline-homepage">
+    <section class="usa-hero" aria-label="Introduction">
+      <div class="grid-container">
+        <div class="usa-hero__callout padding-0">
+          <h1 class="usa-hero__heading font-sans- text-base-darkest">
+            A people-centered platform for online document assembly
+          </h1>
+          <p class="play-video margin-top-0">
+            <a href="#demo-video"
+              class="usa-button text-base-lightest bg-base-darkest" 
+              aria-controls="demo-video"
+              data-open-modal     
+            > 
+              <span class="font-sans-3xs text-normal display-block margin-bottom-1">Play Video | 5 minutes</span>
+              <span class="sans-body-xs display-block">Demo: 10x Access to Justice Forms Platform</span>
+            </a>
+          </p>         
+        </div>
+        <div
+          class="usa-modal usa-modal--lg"
+          id="demo-video"
+          aria-labelledby="modal-1-heading"
+          aria-describedby="modal-1-description"
+        >
+          <div class="usa-modal__content">
+            <div class="usa-modal__main">
+              <div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/d958328ec69548ed99c6cd1a15af8a16?sid=ecc5f80b-931b-403d-addc-538215b53092" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```

Comment:
Can the iframe be loaded when someone clicks the button to open the modal instead of putting into the DOM right away? That way, it will only load for those who intend to play the video. 

I threw together a fiddle with to show how it would work: https://jsfiddle.net/ethangardner/2crkfxsg/

Reply:
Hi @ethangardner . I implemented your suggestion. Thanks for the fiddle.